action.yaml

```diff
@@ -54,6 +54,17 @@ runs:
 
         fi
 
+    - name: Check test_report_path is exists or not
```

Comment:
`is exists` doesn't make much sense. Just `exists`

Reply:
updated